reference/configuration/swiftmailer.rst

```diff
@@ -156,6 +157,13 @@ emails sent during development go to a single account.
 This uses ``Swift_Plugins_RedirectingPlugin``. Original recipients are available
 on the ``X-Swift-To``, ``X-Swift-Cc`` and ``X-Swift-Bcc`` headers.
 
+delivery_whitelist
+~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+If set, only email message send to addresses that match with the values in the list.
```

Comment:
Used in combination with ``delivery_address``. If set, send email as normal to addresses that match with the values in the list. All addresses not matching will be sent to ``delivery_address`` as is the normal behaviour when setting a ``delivery_address``.